Mathlib/Data/Nat/Factorial/Basic.lean

```diff
@@ -12,6 +12,8 @@ import Mathlib.Tactic.Monotonicity.Attr
 # Factorial and variants
 
 This file defines the factorial, along with the ascending and descending variants.
+For the proof that the factorial of `n` counts the permutations of an `n`-element set,
+see `Fintype.card_perm` in `Mathlib.Data.Fintype.Perm`.
```

Comment:
I think file references are a bit too brittle (and our docs linkifier doesn't handle them anyway).
```suggestion
see `Fintype.card_perm`.
```